src/bookshelf/utils.ts

```diff
@@ -72,13 +74,28 @@ function processSample(info: Information, sample: Model): SerialOpts {
         relTemplate.included = false;
     }
 
+    // Add a relation meta function that will add pivot data if it exists
+    relTemplate.relationshipMeta = { data: get(serialOpts, 'relationshipMeta') || relationshipMeta };
+
     template[relName] = relTemplate;
     template.attributes.push(relName);
   });
 
   return template;
 }
 
+function relationshipMeta(relation, models) {
```

Comment:
I would not declare this `relationshipMeta` function here, because it is just the default function to use in case none was passed. 

I would declare it where it is being used as a default.